Задания с 25.02.2022/Task1/Task1/Trie.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Collections.Generic;
+
+/// <summary>
+/// Структура "Бор"
+/// </summary>
+public class Trie
+{
+    private class Node
+    {
+        public Dictionary<char, Node> childs;
+        public bool isTerminal = false;
+
+        internal Node()
+        {
+            childs = new Dictionary<char, Node>();
+        }
+    }
+
+    private Node root = new Node();
```

Comment:
```suggestion
    private Node root = new();
```
В современном C# почти никогда не надо писать тип дважды